qcengine/mdi_server.py

```diff
@@ -83,7 +83,7 @@ def __init__(
         mpi_world = None
         if use_mpi4py:
             mpi_world = MPI.COMM_WORLD
```

Comment:
These lines (83-85) are no longer necessary, and can be removed to resolve an lgtm alert.